help_to_heat/frontdoor/views.py

```diff
@@ -764,15 +796,33 @@ def build_extra_context(self, request, session_id, page_name, data, is_change_pa
         session_data = interface.api.session.get_session(session_id)
 
         address = session_data.get(address_field)
+        country = session_data.get(country_field)
         epc = session_data.get(epc_details_field)
 
         epc_band = epc.get("current-energy-rating")
+        try:
+            epc_date = datetime.strptime(epc.get("lodgement-date"), "%Y-%m-%d")
+            month_name = month_names[epc_date.month - 1]
+            epc_date = epc_date.strftime("%-d") + " " + month_name + " " + epc_date.strftime("%Y")
+        except ValueError:
+            epc_date = epc.get("lodgement-date")
+
+        current_month_start = datetime.now().replace(day=1)
+        month_name = month_names[current_month_start.month - 1]
+        current_month_start = current_month_start.strftime("%-d") + " " + month_name
+
+        next_month_start = (datetime.now() + relativedelta(months=+1)).replace(day=1)
+        month_name = month_names[next_month_start.month - 1]
+        next_month_start = next_month_start.strftime("%-d") + " " + month_name
```

Comment:
Not sure where's a good place to pull this code out to?

Reply:
There's a `utils.py` file (which could probably do with being broken up if we're going to put more into it)